src/providers/StylesProvider.tsx

```diff
@@ -127,6 +127,10 @@ const GlobalStyles = createGlobalStyle(
       height: 100%;
       width: 100%;
       min-height: 814px;
+      
+      @media (max-width: 768px) {
+        min-width: 100vw;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously. -->
🎯 suggestion:
Min-width was set to stop UI collapse for the desktop version.
It seems to me that `min-width: 100vw` is redundant.
Moreover, the vertical scrollbar is not counted in 100vw and it produces an additional horizontal scrollbar now:
![111](https://github.com/user-attachments/assets/f13a0743-7dfb-4534-9bd2-1906b1aa7386)

What do you think about the idea of unsettling min-width instead?